core/src/main/java/org/infinispan/persistence/manager/PersistenceManager.java

```diff
@@ -56,8 +60,6 @@
 
    MarshalledEntry loadFromAllStores(Object key, InvocationContext context);
 
-   void writeToAllStores(MarshalledEntry marshalledEntry, AccessMode modes);
```

Comment:
Please add a `package-info.java` with a `@private` tag, to make it clear this is not part of the API and can change at any time.